plugins/source/aws/resources/services/ses/configuration_sets_fetch.go

```diff
@@ -12,27 +12,35 @@ func fetchSesConfigurationSets(ctx context.Context, meta schema.ClientMeta, pare
 	c := meta.(*client.Client)
 	svc := c.Services().Sesv2
 
-	config := sesv2.ListConfigurationSetsInput{}
-	p := sesv2.NewListConfigurationSetsPaginator(svc, &config)
+	p := sesv2.NewListConfigurationSetsPaginator(svc, nil)
 	for p.HasMorePages() {
 		response, err := p.NextPage(ctx)
 		if err != nil {
 			return err
 		}
 		res <- response.ConfigurationSets
 	}
+
 	return nil
 }
 
 func getConfigurationSet(ctx context.Context, meta schema.ClientMeta, resource *schema.Resource) error {
 	c := meta.(*client.Client)
 	svc := c.Services().Sesv2
-	csName := resource.Item.(string)
+	csName := resource.GetItem().(string)
 
 	getOutput, err := svc.GetConfigurationSet(ctx, &sesv2.GetConfigurationSetInput{ConfigurationSetName: &csName})
 	if err != nil {
 		return err
 	}
-	resource.Item = getOutput
+
+	resource.SetItem(getOutput)
+
 	return nil
 }
+
+func resolveConfigurationSetArn(ctx context.Context, meta schema.ClientMeta, resource *schema.Resource, c schema.Column) error {
```

Comment:
Can we do this change in another PR? Adding resources using codegen already generates quite a lot of changes to review, so changing primary keys for resources not added makes it harder to review the PR